gmso/core/views.py

```diff
@@ -0,0 +1,143 @@
+from collections import defaultdict
+
+from gmso.core.angle import Angle
+from gmso.core.angle_type import AngleType
+from gmso.core.atom import Atom
+from gmso.core.atom_type import AtomType
+from gmso.core.bond import Bond
+from gmso.core.bond_type import BondType
+from gmso.core.dihedral import Dihedral
+from gmso.core.dihedral_type import DihedralType
+from gmso.core.improper import Improper
+from gmso.core.improper_type import ImproperType
+
+potential_attribute_map = {
+    Atom: "atom_type",
+    Bond: "bond_type",
+    Angle: "angle_type",
+    Dihedral: "dihedral_type",
+    Improper: "improper_type",
+}
+
+
+def get_identifier(potential):
```

Comment:
We can potential condense some of these identifiers (see my other suggested ones below) by adding some flags to this function. Or we can rename this function to "get_parameter_names" which may be more indicative of what this function is doing.